doc/autoscale/experiment/README.md

```diff
@@ -1,26 +1,40 @@
 # Auto-scaling Experiment Design
 
+## Purpose
 
-## Environment Enviroment
+To verify effectiveness of PaddlePaddle's fault-tolerance and auto-scaling mechanism.
```

Comment:
-> "To verify the effectiveness of PaddlePaddle's fault-tolerance and auto-scaling mechanism."